man/systemd.network.xml

```diff
@@ -1618,6 +1618,13 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>MTUBytes=</varname></term>
+          <listitem>
+            <para>As in [Route] section.</para>
```

Comment:
please say that this changes the MTU for the routes DHCP defines, and then point the user to the [Route] section for further details. i.e. it's key to say this is the route MTU, and not an interface MTU or so